lib/common/http/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { log as logger } from '../../logs/logger';
+export const switchToInsecure = (url: string) => {
+  logger.debug({ url }, 'Forcing insecure url');
+  const urlToSwitch = new URL(url);
+  urlToSwitch.protocol == 'http';
```

Comment:
@aarlaud, I think it should be assignment instead of comparison
```
urlToSwitch.protocol = 'http';
```

Reply:
:facepalm: fixed